### ISSUE TITLE
Support for Lyrics - Web UI

### ISSUE THREAD

**Issue by iWangJiaxiang** [navidrome/navidrome#491]
It would be nice to support lyric display feature

**Comment by deluan** [comment]
Agree. Both the React Player used in Navidrome, as well as the Subsonic API, have provisions to expose lyrics. I'll implement this at some point, before v1.0

**Comment by iWangJiaxiang** [comment]
This project [MoePlayer/APlayer: Wow, such a beautiful HTML5 music player](https://github.com/MoePlayer/APlayer) has lyric support.

I believe the frontend tech is enough for lyric display and scrolling right now~

**Comment by githubA3B5** [comment]
Is it a good way for getting lyric sources with custom http url or plugins way ?

**Comment by deluan** [comment]
Sorry @githubA3B5 , I didn't understand what you mean... Currently Navidrome does not support lyrics, but as the upstream [React Player](https://github.com/lijinke666/react-music-player) has support for it, I'll implement it at some point

**Comment by rohitgeddam** [comment]
I see that Airsonic has lyrics support, maybe we can see how they implemented it.

**Comment by deluan** [comment]
Closing this in favour of #249

**Comment by joseph-cheung** [comment]
How is it going? Does it now support the display of lrc format lyrics files?

**Comment by deluan** [comment]
> How is it going? Does it now support the display of lrc format lyrics files?

@dheerajdlalwani recently started working on adding support for lyrics (both sync and unsync). We will probably support it first in the Subsonic API (#249), then move to implement sync lyrics in the UI. Unsync lyrics may take longer to get to the UI, as the current player we use does not support it

**Comment by deluan** [comment]
This was implemented in #1406. Only synced lyrics (lrc format) are supported but they have to be embedded as a tag in the music file. Support for external files (`.lrc`) will be added later.
